src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/lib/processor_types.js

```diff
@@ -0,0 +1,12 @@
+export default [
```

Comment:
As I mentioned on processor.js, you *could* make this an actual subclass of Processor. It would make the relationship between the two a bit less awkward and adhoc.